src/Tribe/Service_Providers/First_Boot.php

```diff
@@ -60,6 +60,16 @@ public function set_default_options() {
 			$options['tribeEventsTemplate'] = '';
 		}
 
+		if ( ! isset( $options['tribeEnableViews'] ) ) {
+			// Set the Enabled Views to List, Month, and Day
+			$options['tribeEnableViews'] = [ 'list', 'month', 'day' ];
+		}
+
+		if ( ! isset( $options['viewOption'] ) ) {
+			// Set the Default View to List
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: b95d404dd688c37720f8194e4b476d95</sub>